cmd/containerd/main.go

```diff
@@ -148,10 +148,13 @@ func main() {
 }
 
 func before(context *cli.Context) error {
-	if err := loadConfig(context.GlobalString("config")); err != nil &&
-		!os.IsNotExist(err) {
+	err := loadConfig(context.GlobalString("config"))
+	if err != nil && !os.IsNotExist(err) {
 		return err
+	} else if err != nil && os.IsNotExist(err) {
+		log.G(global).Warnf("config %q does not exist to be loaded", context.GlobalString("config"))
```

Comment:
oh? perhaps with the other PR for adding a `config.WriteTo()`, then if it doesn't exist, just write the default one?